src/ui/component/publishName/bid-help-text.jsx

```diff
@@ -0,0 +1,32 @@
+// @flow
+type Props = {
+  uri: ?string,
+  isResolvingUri: boolean,
+  amountNeededForTakeover: number,
+};
+
+function BidHelpText(props: Props) {
+  const { uri, isResolvingUri, amountNeededForTakeover } = props;
+
+  let bidHelpText;
+
+  if (uri) {
+    if (isResolvingUri) {
+      bidHelpText = __('Checking the winning claim amount...');
+    } else if (!amountNeededForTakeover) {
+      bidHelpText = __('Any amount will give you the winning bid.');
+    } else {
+      bidHelpText = `${__('If you bid more than')} ${amountNeededForTakeover} LBC, ${__(
```

Comment:
`@i18nfixme`

(also, these split strings are basically not worth even calling the function on, but should be tagged either way)